--- /src/System.cc      2018-01-23 19:34:08.091713831 +0000
+++ /src/System.cc      2018-01-23 19:22:11.336399989 +0000
@@ -23,7 +23,6 @@
 #include "System.h"
 #include "Converter.h"
 #include <thread>
-#include <pangolin/pangolin.h>
 #include <iomanip>

 namespace ORB_SLAM2
@@ -94,14 +93,6 @@
     mpLoopCloser = new LoopClosing(mpMap, mpKeyFrameDatabase, mpVocabulary, mSensor!=MONOCULAR);
     mptLoopClosing = new thread(&ORB_SLAM2::LoopClosing::Run, mpLoopCloser);

-    //Initialize the Viewer thread and launch
-    if(bUseViewer)
-    {
-        mpViewer = new Viewer(this, mpFrameDrawer,mpMapDrawer,mpTracker,strSettingsFile);
-        mptViewer = new thread(&Viewer::Run, mpViewer);
-        mpTracker->SetViewer(mpViewer);
-    }
-
     //Set pointers between threads
     mpTracker->SetLocalMapper(mpLocalMapper);
     mpTracker->SetLoopClosing(mpLoopCloser);
@@ -315,8 +306,6 @@
         usleep(5000);
     }

-    if(mpViewer)
-        pangolin::BindToContext("ORB-SLAM2: Map Viewer");
 }

 void System::SaveTrajectoryTUM(const string &filename)
